src/main/kotlin/no/nav/familie/ef/sak/service/TilgangService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.ef.sak.service
+
+import no.nav.familie.ef.sak.api.ManglerTilgang
+import no.nav.familie.ef.sak.integration.FamilieIntegrasjonerClient
+import no.nav.familie.ef.sak.sikkerhet.SikkerhetContext
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class TilgangService(private val integrasjonerClient: FamilieIntegrasjonerClient,
+                     private val personService: PersonService,
+                     private val behandlingService: BehandlingService,
+                     private val fagsakService: FagsakService) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun validerTilgangTilPersonMedBarn(personIdent: String) {
+        val person = personService.hentPersonMedRelasjoner(personIdent)
+
+        integrasjonerClient.sjekkTilgangTilPersoner(person.identifikatorer()).forEach {
```

Comment:
person.identifikatorer() - er dette søkers identifikatorer, eller er barn også med her?

Reply:
Bern er også med

Reply:
Fant svaret: fun identifikatorer(): List<String> = listOf(søkerIdent) + barn.keys
identifikatorer -> bedre navn?

Reply:
Holder det med barn? Burder ikke foreldrene også være med. RealterteFnr som ligger på søknad er et sett med fødselnummerne til alle barn og foreldre fra søknaden.

Reply:
Sjekker dere spørsmålet til Bent med funksjonelle?

Reply:
Uansett utfall av prat med funksjonelle tror jeg litt bedre navngivning av identifikatorer kunne vært bra?
Evt. gjøre opprettelse av disse listene mer eksplisitt. Hente id'er for søker, hente id'er for barn, (evt. andre foreldre ;) ) 
og lage listen her?

Reply:
Jeg stilte det spørsmålet videre til Mirja for en tid tilbake, men har ikke fått noe svar.
Her sjekker vi personen mot pdl og finner alle barna derfra pdl. 
Kanskje burde vi brukt "søknaden", men hva skjer når du skal hente alle behandlinger/fagsaker  - da må vi iterere igjennom alle og sjekke alle barn og foreldre? 

I Journalposten har vi ikke søknaden "lett tilgjengelig" i kode og PDL-data virker plausibelt nok.

Reply:
Point. Dette er en liten nøtt -> men jo mer jeg tenker på det...
Saksbehandler skal jo vurdere mange ting hvor vi må bruke data om den andre forelderen? Som bla.a. hvor den andre forelderen bor. 

Kanskje vi ikke trenger sjekke dette her/nå, men det må sikkert sjekkes på ett eller annet tidspunkt?  Når?

Reply:
Nå bare babler jeg: men er det mulig å sjekke andre foreldre (som registrert i pdl) her og ta en ekstra sjekk av hva som ligger i søknad senere?

Reply:
Ja, vi kan vel få ut "andre" foreldre fra pdl og sjekke på disse i tillegg. Da dekker vi bredt nok, men potensielt "for bredt" for f.eks. barnetilsyn.

Reply:
Du du kan ta med relaterte fnr når du spår etter barna fra pdl. Det vil normalt bare være fnr til søker og ev. fnr til den andre forelderen.